workers/unity/Assets/Gdk/Core/MutableView/MutableView.cs

```diff
@@ -13,16 +13,12 @@ namespace Improbable.Gdk.Core
     public class MutableView
     {
         public const long WorkerEntityId = -1337;
```

Comment:
This can go away right?

Reply:
Not unless we want magic numbers somewhere